docs/en/latest/plugins/fault-injection.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: fault-injection
+keywords:
+  - APISIX
+  - Plugin
+  - Fault Injection
+  - fault-injection
+description: This document contains information about the Apache APISIX fault-injection Plugin.
```

Comment:
The skywalking-eye failed because you added **too much** before the license header, please take a look at this (I admit I forgot to document this, will add it in our index doc)

https://github.com/apache/skywalking-eyes/blob/66a8bd6aa7e8246c061b2d9baf7ebe646f9eb475/.licenserc.yaml#L79-L81

Reply:
@kezhenxu94 Does the 80 represent number of lines? If so, updating our configuration file to check for more lines should work right?

Reply:
> @kezhenxu94 Does the 80 represent number of lines? If so, updating our configuration file to check for more lines should work right?

It’s not line count, it’s character count 🥲

Reply:
Oh. We will put some bigger number and try.